app/models/movie.rb

```diff
@@ -1,5 +1,15 @@
 class Movie < ApplicationRecord
+  belongs_to :text
```

Comment:
デフォルトで `presence: true` が入るのでエラーが出ていますね。

```suggestion
  belongs_to :text, optional: true
```